packages/astro/src/content/runtime.ts

```diff
@@ -130,6 +176,14 @@ export function createGetCollection({
 				liveCollections[collection].loader as LiveLoader<any, any, Record<string, unknown>>
 			)?.loadCollection?.(context);
 
+			const { schema } = liveCollections[collection];
+
+			if (schema) {
```

Comment:
Should we check if live collections are enabled?

Reply:
Same as above